pkgs/applications/window-managers/notion/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  enableXft ? true, fontconfig ? null,
+  patches ? [],
+  fallbackFont ? "Sans",
+  stdenv, fetchurl,
+  lua, gettext, groff,
+  pkgconfig, busybox,
+  x11, libXinerama, libXrandr, libX11
+}:
+
+assert enableXft -> fontconfig != null && fallbackFont != null;
+
+let
+  xftPatch = "./notion-xft_nixos.diff";
+  name     = "notion";
```

Comment:
Put name and version directly in the derivation, it's hard to read with a let expression.